app/users/update-user/updateUserInteractor.js

```diff
@@ -26,7 +33,15 @@ class UpdateUserInteractor {
             return;
         }
 
-        const userEntity = this.userFactory.create({ id: request.id, name: request.name });
+        let password;
+        if (!request.password) {
+            password = user.getPassword();
+        } else {
+            const SALT_ROUNDS = 11;
+            password = await this.bcrypt.hash(request.password, SALT_ROUNDS);
+        }
```

Comment:
To change password we need separate root, since it is not common case to change it within such data as email or name, etc.
For now, you can ignore adding `/change-password` since it has no use for this PR

Reply:
But remove this functionality)